core/src/test/scala/integration/kafka/api/BaseProducerSendTest.scala

```diff
@@ -227,26 +227,6 @@ abstract class BaseProducerSendTest extends KafkaServerTestHarness {
     }
   }
 
-  @Test
-  def testWrongSerializer() {
```

Comment:
Why are we removing the test? It is a condition that cannot happen with new producer?